tests/test_roxarapi/test_roxarapi_reek.py

```diff
@@ -186,6 +186,53 @@ def test_rox_surfaces_alternative_open(roxar_project):
     rox.project.close()
 
 
+@pytest.mark.requires_roxar
+def test_rox_surfaces_clipboard_general2d_data(roxar_project, roxinstance):
+    """Set and get surfaces on clipboard and general2D data"""
+
+    rox = xtgeo.RoxUtils(roxar_project)
+    project = rox.project
+
+    surf = xtgeo.surface_from_roxar(project, "TopReek", SURFCAT1)
+
+    surf.to_roxar(project, "mycase", "myfolder", stype="clipboard")
+    surf2 = xtgeo.surface_from_roxar(project, "mycase", "myfolder", stype="clipboard")
+    assert surf2.values.mean() == surf.values.mean()
+
+    # general 2D data (from xtgeo version 2.19 and roxar API >= 1.6)
+    if not roxinstance.version_required("1.6"):
+        with pytest.raises(
+            NotImplementedError, match=r"API Support for general2d_data is missing"
+        ):
+            surf.to_roxar(project, "mycase", "myfolder", stype="general2d_data")
+        logger.info("This version of RMS does not support this feature")
+
+    else:
+        surf.to_roxar(project, "mycase", "myfolder", stype="general2d_data")
+        surf2 = xtgeo.surface_from_roxar(
+            project, "mycase", "myfolder", stype="general2d_data"
+        )
+        assert surf2.values.mean() == surf.values.mean()
```

Comment:
I suggest `surf2.values.tolist() == surf.values.tolist()` so that all values are checked.